flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
This is because currently the upstream CI only builds for x86. While I believe it's trivial to add support for arm64 with Flutter, a new build is still needed and I do not expect it to arrive soon.

Reply:
Now it should build for both `x86_64` and `aarch64`.